var/spack/repos/builtin/packages/cblas/package.py

```diff
@@ -42,17 +42,17 @@ class Cblas(Package):
     def patch(self):
         mf = FileFilter('Makefile.in')
 
-        mf.filter('^BLLIB =.*', 'BLLIB = %s/libblas.a' %
-                  self.spec['blas'].prefix.lib)
+        mf.filter('^BLLIB =.*', 'BLLIB = {0}'.format(
+                  ' '.join(self.spec['blas'].libs.libraries)))
         mf.filter('^CC =.*', 'CC = cc')
-        mf.filter('^FC =.*', 'FC = f90')
+        mf.filter('^FC =.*', 'FC = fc')
 
     def install(self, spec, prefix):
         make('all')
         mkdirp(prefix.lib)
         mkdirp(prefix.include)
 
         # Rename the generated lib file to libcblas.a
-        install('./lib/cblas_LINUX.a', '%s/libcblas.a' % prefix.lib)
-        install('./include/cblas.h', '%s' % prefix.include)
-        install('./include/cblas_f77.h', '%s' % prefix.include)
+        install('lib/cblas_LINUX.a', prefix.lib.join('libcblas.a'))
```

Comment:
not that this should be blocking here, but: What about OSX? Will it also be `cblas_LINUX.a`?

Reply:
Yep, the `PLAT` is hardcoded to `LINUX`. Honestly, it might be better to rename the library that gets created in the Makefile instead of after creation, but I don't know if I care enough about this package. As long as it works.